libs/vscode/nx-help-and-feedback-view/src/lib/nx-help-and-feedback-provider.ts

```diff
@@ -42,4 +111,84 @@ export class NxHelpAndFeedbackProvider extends AbstractTreeProvider<NxHelpAndFee
       ([title, link, icon]) => new NxHelpAndFeedbackTreeItem(title, link, icon)
     );
   }
+
+  async isConnectedToCloud(): Promise<boolean> {
+    if (!this.nxConfig?.tasksRunnerOptions) {
+      return false;
+    }
+    return !!Object.values(this.nxConfig.tasksRunnerOptions).find(
+      (r) => r.runner == '@nrwl/nx-cloud'
+    );
+  }
+
+  async connectToCloud() {
+    getTelemetry().featureUsed('nxConsole.connectToNxCloud');
+
+    const isConnected = await this.isConnectedToCloud();
+    if (isConnected) {
+      window.showInformationMessage('You are already connected to Nx Cloud');
+      return;
+    }
+
+    const nxVersion = await WorkspaceConfigurationStore.instance.get(
+      'nxVersion',
+      'latest'
+    );
+
+    window.withProgress(
+      {
+        location: ProgressLocation.Notification,
+        title: 'Connecting you to Nx Cloud...',
+      },
+      () => {
+        return new Promise((resolve) => {
+          try {
+            if (lt(coerce(nxVersion) ?? '', '12.0.0')) {
```

Comment:
This is just a note to self: move the version parser in a util function to use for other packages (like lerna)